subprojects/core/src/main/java/org/gradle/configuration/BuildTreePreparingProjectsPreparer.java

```diff
@@ -75,6 +75,7 @@ private void generateDependenciesAccessorsAndAssignPluginVersions(ServiceRegistr
         dm.getDefaultLibrariesExtensionName().finalizeValue();
         String defaultLibrary = dm.getDefaultLibrariesExtensionName().get();
         File dependenciesFile = new File(settings.getSettingsDir(), "gradle/libs.versions.toml");
+        // TODO:configuration-cache causes test failures (see "configuration cache for help on empty project")
```

Comment:
:question: I'm not sure I understand, could you clarify that comment?

Reply:
I'm not even sure that this line is right... we've been digging around the version catalogs with Rodrigo, and found some issues (e.g. here, where CC doesn't know that we try to open the default VC).

I think I will try to instrument this, and see if it behaves as expected.